.env

```diff
@@ -21,3 +21,11 @@ WEBAPP_URL=http://localhost:8000/
 API_URL=/api/v1/
 TEMPORAL_HOST=airbyte-temporal:7233
 INTERNAL_API_HOST=airbyte-server:8001
+
+# Fill the below variables to log to cloud storage. The provided credentials require both S3 read/write permissions. The logger attempts to create the bucket if
```

Comment:
Are we going to leave this for docker compose?

Reply:
yeah. you think it's cleaner to remove and have this only in documentation?

Reply:
done